python/lsst/ts/ess/ess_csc.py

```diff
@@ -60,9 +152,7 @@ def get_config_pkg():
     @classmethod
     def add_arguments(cls, parser):
         super(EssCsc, cls).add_arguments(parser)
-        parser.add_argument(
-            "-s", "--simulate", action="store_true", help="Run in simuation mode?"
-        )
+        parser.add_argument("-s", "--simulate", action="store_true", help="Run in simuation mode?")
```

Comment:
See below. This is black vs. flake8 formatting. I would rather drop this  change and switch to black formatting.

Reply:
Yes I still agree and as mentioned above I'd like to do that in a separate ticket. And as stated as before, the code was formatted by black via a githook and an IDE plugin and the line length used to be set to 80 and now was changed to 110 so that's why these lines were automatically reformatted.